modules/drivers/presto-jdbc/src/metabase/driver/presto_jdbc.clj

```diff
@@ -131,6 +131,10 @@
   (sql.qp/cast-temporal-string driver :Coercion/YYYYMMDDHHMMSSString->Temporal
                                [:from_utf8 expr]))
 
+(defmethod sql.qp/->honeysql [:presto-jdbc ::sql.qp/cast-to-text]
+  [driver [_ expr]]
+  (h2x/maybe-cast "varchar" (sql.qp/->honeysql driver expr)))
```

Comment:
Is there a reason this one isn't implemented in terms of `::sql.qp/cast` like the others?